examples/gravitational_waves_detection.ipynb

```diff
@@ -0,0 +1,438 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Gravitational wave detection\n",
+    "Chrisopher Bresten and Jae-Hun Jung propose to include topological features in a CNN classifier for gravitational waves detection. Adapted from their article, this notebook showcases an application of ideas from the [Topology of time series example](https://giotto-ai.github.io/gtda-docs/latest/notebooks/topology_time_series.html).\n",
```

Comment:
Suggestion: `[Topology of time series](https://giotto-ai.github.io/gtda-docs/latest/notebooks/topology_time_series.html) example`